packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/MrwSerializationTypeDefinitions/JsonModelCoreTests.cs

```diff
@@ -42,10 +44,11 @@ public void KebabCaseSerializedName()
                 "kebab-case",
                 "kebab-case",
                 "A property with kebab-case name",
-                new InputPrimitiveType(InputPrimitiveTypeKind.String, null),
+                new InputPrimitiveType(InputPrimitiveTypeKind.String),
                 true,
                 false,
-                false);
+                false,
+                Array.Empty<InputDecoratorInfo>());
```

Comment:
we could just use `[]` here.
.Net 8 supports collection expressions, and it is recommended over other explicit API calls.
The compiler will find the most sufficient way to construct the collection in the current context so that we do not need to worry about it.